plugins/modules/openvswitch_db.py

```diff
@@ -0,0 +1,238 @@
+#!/usr/bin/python
+# coding: utf-8 -*-
+
+#
+# (c) 2015, Mark Hamilton <mhamilton@vmware.com>
+# Portions copyright @ 2015 VMware, Inc.
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
```

Comment:
This repo is Apache licensed. How does it play with mixing in GPL code, 3.0 at that?

Reply:
This is what Apache has to say on the matter[0]:

> Apache 2 software can therefore be included in GPLv3 projects, because the GPLv3 license accepts our software into GPLv3 works. However, GPLv3 software cannot be included in Apache projects. The licenses are incompatible in one direction only, and it is a result of ASF's licensing philosophy and the GPLv3 authors' interpretation of copyright law.

Seems like this would be one of the cases when it's not possible.

[0] https://www.apache.org/licenses/GPL-compatibility.html